src/Npgsql/Internal/TypeMapping/TypeMapperResolver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Npgsql.Internal.TypeHandling;
+using Npgsql.PostgresTypes;
+
+namespace Npgsql.Internal.TypeMapping;
+
+public abstract class TypeMapperResolver
```

Comment:
The TypeMapperResolver name seems a bit confusing - it doesn't resolve TypeMappers, but rather TypeMappings... Maybe TypeMappingResolver?